graphql2/graphqlapp/messagelog.go

```diff
@@ -211,6 +212,9 @@ func (q *Query) MessageLogs(ctx context.Context, opts *graphql2.MessageLogSearch
 		if log.ProviderMsgID != nil {
 			dm.ProviderID = &log.ProviderMsgID.ExternalID
 		}
+		if log.SentAt != nil {
+			dm.SentAt = log.SentAt
+		}
```

Comment:
These can both be nil, so the `if` check isn't necessary.
```suggestion
		dm.SentAt = log.SentAt
```

Reply:
Moved the setting of dm.SentAt into the struct instantiation.